vic/drivers/cesm/bld/vic.parameters.txt

```diff
@@ -0,0 +1,211 @@
+#######################################################################
+# VIC Model Parameters
+#######################################################################
+
+# Lapse Rate
```

Comment:
There seems to be a lot of duplications with `vic/drivers/cesm/bld/vic.constants.txt`. Shouldn't they be specified in one or the other, but not both? Also update comment to make clear what the difference is (for this purpose) between a `parameter` and a `constant.